cl/citations/match_citations.py

```diff
@@ -83,26 +94,23 @@ def case_name_query(conn, params, citation, citing_doc):
     return results
 
 
-def get_years_from_reporter(citation):
+def get_years_from_reporter(citation: CitationBase) -> Tuple[int, int]:
     """Given a citation object, try to look it its dates in the reporter DB"""
     start_year = 1750
     end_year = date.today().year
-    if citation.lookup_index is not None:
-        # Some cases can't be disambiguated.
-        # fmt: off
-        reporter_dates = (REPORTERS[citation.canonical_reporter]
-                          [citation.lookup_index]
-                          ['editions']
-                          [citation.reporter])
-        # fmt: on
-        if hasattr(reporter_dates["start"], "year"):
-            start_year = reporter_dates["start"].year
-        if hasattr(reporter_dates["end"], "year"):
-            end_year = reporter_dates["end"].year
+
+    edition_guess = citation.edition_guess
```

Comment:
I'm not entirely sure this is correct, so I want to flag this as requiring a closer look. The previous logic with the four-levels of nesting in the `REPORTERS` variable confused me, but I think we can just grab the same information now using the `edition_guess` attribute?

Reply:
This looks right to me.